etc/config-template.toml

```diff
@@ -96,7 +96,7 @@
 # grpc-memory-pool-quota = "32G"
 
 ## The number of connections with each TiKV server to send Raft messages.
-# grpc-raft-conn-num = 10
+# grpc-raft-conn-num = 1
```

Comment:
why do you change this to one?

Reply:
@shenli I saw that this default value is actually 1 in our code